docs/sources/tempo/release-notes/v2-3.md

```diff
@@ -0,0 +1,201 @@
+
+Tempo 2.3 makes [vParquet3]({{< relref "../configuration/parquet" >}}), a Parquet version designed to be more compatible with other Parquet implementations, available as a production-ready option.
+This block format is required for using structural operators and improves query performance relative to previous formats.
+
+Read the [**Tempo 2.3 blog post**](NEED-LINK) for more examples and details about these improvements.
+
+These release notes highlight the most important features and bugfixes. For a complete list, refer to the [Tempo changelog](https://github.com/grafana/tempo/releases).
+
+## Features and enhancements
+
+The most important features and enhancements in Tempo 2.3 are highlighted below.
+
+### vParquet3 promoted to stable
+
+[vParquet3]({{< relref "../configuration/parquet" >}}), an updated Parquet block format introduced in Tempo 2.2, has been promoted to stable and is production ready.
+We're excited about vParquet3 relative to prior formats because of its support for dedicated attribute columns, which help speed up queries on your most queried attributes. [[PR 2649](https://github.com/grafana/tempo/pull/2649)]
+
+To read more about the design of vParquet3, refer to [the design proposal](https://github.com/grafana/tempo/blob/main/docs/design-proposals/2023-05%20vParquet3.md).
+For general information, refer to [the Apache Parquet schema]({{< relref "../operations/schema" >}}).
+
+#### Dedicated attribute columns
+
+Dedicated attribute columns improve query performance by storing the most frequently used attributes in their own columns, rather than in the generic attribute key-value list.
+
+Introduced with vParquet3, dedicated attribute columns are only available when using this storage format.
+
+To learn more about this new feature, refer to the [Dedicated attribute columns]({{< relref "../operations/dedicated_columns" >}}) documentation.
+
+### TraceQL
+
+Unique to Tempo, TraceQL is a query language that lets you perform custom queries into your tracing data.
+To learn more about the TraceQL syntax, see the [TraceQL documentation]({{< relref "../traceql" >}}).
+For information on planned future extensions to the TraceQL language, refer to [future work]({{< relref "../traceql/architecture" >}}).
+
+We’ve made the following improvements to TraceQL:
+
+* Added two structural operators, ancestor (`<<`) and parent (`<`) ([documentation]({{< relref "../traceql#experimental-structural" >}})) [[PR 2877](https://github.com/grafana/tempo/pull/2877)]
+
+* Added experimental not structural operators: `!>`, `!<`, and `!~` ([documentation]({{< relref "../traceql#experimental-structural" >}})) [[PR 2993](https://github.com/grafana/tempo/pull/2993)]
+
+* Added quoted attribute syntax so you can search for attributes with terminal characters like periods (`.`) in their names ([documentation]({{< relref "../traceql#experimental-structural" >}})) [[PR 3004](https://github.com/grafana/tempo/pull/3004)]
```

Comment:
periods were always fine. i think space ` ` is the most obvious character we've enabled the use of

Reply:
```suggestion
* Added quoted attribute syntax so you can search for attributes with terminal characters like spaces in their names ([documentation]({{< relref "../traceql#experimental-structural" >}})) [[PR 3004](https://github.com/grafana/tempo/pull/3004)]
```

Reply:
here is the list of characters we previously terminated an attribute name on in traceql. if these characters showed up in an attribute name they would have correctly returned when retrieving the trace by id but they were not referenceable in traceql:

[whitespace chars](https://github.com/grafana/tempo/blob/main/pkg/traceql/lexer.go#L345)
[other chars](https://github.com/grafana/tempo/blob/main/pkg/traceql/lexer.go#L350)

maybe we just say "space and other special chars?"